.github/workflows/nv-pre-compile-ops.yml

```diff
@@ -7,8 +7,6 @@ on:
     paths-ignore:
       - 'docs/**'
       - 'blogs/**'
-      - 'deepspeed/inference/v2/**'
-      - "tests/unit/inference/v2/**"
```

Comment:
I think we want to keep these since the pre-compile-ops do not test anything related to inference v2

Reply:
Yep, forgot to add those back in, should be fixed now.